src/Serializable.ts

```diff
@@ -165,10 +217,24 @@ export class Serializable extends SerializableBase implements ISerializable {
         return this.postFrom(realObj)
     }
 
+    /**
+     * `preFrom` can be overwritten to manipulate the value before the parsing.
+     * This allows to add logic to the deserialization without having to override the fromAny method.
```

Comment:
This allows to add logic to the **parsing** without having to override the fromAny method.